src/graphql/schema.js

```diff
@@ -239,11 +239,30 @@ type PaginatedBiggestWinner {
   items: [BiggestWinner]!
 }
 
+type Leaderboard {
+  eventAddress: String
+  userAddress: String!
+  investments: String!
+  winnings: String!
+  returnRatio: String!
```

Comment:
change to Int

Reply:
Think percentage should be a float number right? I can change it to Float

Reply:
sorry ya Float, not Int. we want to retain 2 decimal places.